dotCMS/src/main/java/com/dotcms/contenttype/transform/field/JsonFieldTransformer.java

```diff
@@ -149,6 +152,12 @@ public JSONArray jsonArray() {
       field.remove("acceptedDataTypes");
       field.remove("dbColumn");
 
+      if (ImmutableCategoryField.class.getName().equals(field.get("clazz"))) {
```

Comment:
is this better than a `if(field instanceof CategoryField)` ?

Reply:
really field is a map